serverless.example.yml

```diff
@@ -61,19 +72,25 @@ functions:
   last_transactions:
     handler: src/last-transactions.lastTransactions
     timeout: 30
-    memorySize: 256
+    memorySize: 128
```

Comment:
I see you decreased the `memorySize`, `ttlInSeconds` and `schedule`. Are the new values expected?

Reply:
`ttlSeconds` is subjective :shrug: it defines the cache timeout. The longer the cache timeout the less these functions will cost us.
`schedule` is also arguably subjective, but they want a high resolution history of prices, so I figured one minute was ideal.
The functions work fine with 128 MB of memory and it naturally costs us less. These functions are predominately I/O bound.